### ISSUE TITLE
Multiple errors when installing docz

### ISSUE THREAD

**Issue by lceconi** [doczjs/docz#1735]
## Bug Report

**Describe the bug**

I'm getting multiple errors when following the installation instructions in docz documentation.

**To Reproduce**

1. Inside an empty folder, run `yarn add docz react react-dom`
2. Once the install is finished, add the following scripts to you `package.json`:
```
"scripts": {
    "docz:dev": "docz dev",
    "docz:build": "docz build",
    "docz:serve": "docz build && docz serve"
  }
```
3. Run `yarn docz:dev`
4. See the first error `Error: Cannot find module 'typescript'`
5. Install Typescript: `yarn add typescript`
6. Run `yarn docz:dev` again
7. See error `Can't resolve '@emotion/core' in...`
8. Install emotion: `yarn add @emotion/core`
9. Run `yarn docz:dev` again
10. More errors: `Error [ERR_REQUIRE_ESM]: Must use import to load ES Module`
![image](https://user-images.githubusercontent.com/4535535/175538973-57417fb4-49e4-49e0-89eb-2c07de217669.png)

![image](https://user-images.githubusercontent.com/4535535/175539082-7c78c12e-15ee-448c-a7da-7a553a8555f1.png)

**Expected behavior**

To run `yarn docz:dev` and `yarn docz:serve` without errors.

**Environment**

- docz version: 2.4.0
- OS: OSX 12.3.1
- Node/npm version: Node v14.18.3/npm 6.14.15

**Comment by Xiphe** [comment]
Can confirm this issue

**Comment by lceconi** [comment]
I was able to mitigate this problem by installing specific lib versions. So, to properly run docz in development run:
`yarn add docz react@16.8.0 react-dom@16.8.0 typescript @emotion/core@10.1.1`

Then add the scripts in your `package.json`:
```
"scripts": {
  "docz:dev": "docz dev",
  "docz:build": "docz build",
  "docz:serve": "docz build && docz serve"
}
```

And run `yarn docz:dev`. Now you can navigate to `localhost:3000` to see it running.

**Comment by Xiphe** [comment]
Ah thanks! I eventually gave up on docz and am evaluating https://github.com/facebook/docusaurus now

**Comment by rozzzly** [comment]
> Ah thanks! I eventually gave up on docz and am evaluating https://github.com/facebook/docusaurus now

Same. Docz appears to be dead/unmaintained. Numerous issues (#1719, #1731, #1674) where the "fix" is to pin some dep like a 2019 react release isn't a real solution. To be fair, a large part of this is because of upstream issues with various deps.

#1634 appears to indicate one of the maintainers is no longer active. I would suggest adding a note at the top of the `README.md` making it obvious that there's not active development/support.

**Comment by pedronauck** [comment]
cc @renatobenks

**Comment by pedronauck** [comment]
@renatobenks is the actual maintainer of the project guys, he can talk better about maintaining

**Comment by Xiphe** [comment]
Thanks @pedronauck and big thanks anyone who put work into this project. I know the burden of keeping stuff up to date and running myself.

**Comment by calebpanza** [comment]
Wondering if there's any update on this. Trying to avoid using Storybook if possible cause it can be such a headache. I LOVE the setup of docz but won't be able to use it if this problem persists...

**Comment by Xiphe** [comment]
Hey @calebpanza 👋 as mentioned above, you might want to give [docusaurus](https://github.com/facebook/docusaurus) a shot. Or [react styleguidist](https://react-styleguidist.js.org/)

**Comment by Xiphe** [comment]
Would anyone be interested in [rebuilding this project](https://github.com/doczjs/docz/discussions/1740)?

**Comment by Xiphe** [comment]
No, have not used Docusaurus in production yet. But it sticked out when I was researching alternatives to storybook and docz.

**Comment by Ghullam87** [comment]
### **[✔✔✨ Download IDM Crack For Mac](https://axcrack.org/dl/)**
### **[✔✔✨ Download IDM Crack For Windows](https://axcrack.org/dl/)**

**Comment by angeshsingh111** [comment]
**Download Linkk👇 👇**

### Server 1 =► [https://bit.ly/Full_Movie-Watch](https://bit.ly/Full_Movie-Watch)
### Server 2 =► [https://bit.ly/Full_Movie-Watch](https://bit.ly/Full_Movie-Watch)
### Server 3 =► [https://bit.ly/Full_Movie-Watch](https://bit.ly/Full_Movie-Watch)
### Server 4 =► [https://bit.ly/Full_Movie-Watch](https://bit.ly/Full_Movie-Watch)

**Comment by ramazansancar** [comment]
https://github.com/doczjs/docz/issues/1735#issuecomment-1202845775 @lauraceconi 

Unfortunately, these didn't work either.
